src/servers/src/mysql/handler.rs

```diff
@@ -97,7 +97,14 @@ impl MysqlInstanceShim {
         {
             vec![Ok(output)]
         } else {
-            self.query_handler.do_query(query, query_ctx).await
+            let trace_id = query.len() as u64;
+            common_telemetry::info!("trace_id: {}, query: {}", trace_id, query);
+
+            common_telemetry::TRACE_ID
+                .scope(trace_id, async move {
+                    self.query_handler.do_query(query, query_ctx).await
+                })
+                .await
```

Comment:
Are these test codes? 😏

Reply:
Sure! I've changed it to the real id get from ctx